MonoDevelop.MSBuild.Editor/MSBuildContentType.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
```

Comment:
I don't think this is needed? It'll likely be grayed out by code analysis

Reply:
Yes. Removed it